### PR TITLE
API: removal of drf_yasg (OpenAPI 2.0 Swagger)

### PR DESCRIPTION
**Pull request by kiblik**

### CODE REVIEW

**Comment by mtesauro** [comment]
@kiblik Mind looking at those 2 minor Flake 8 issues? Otherwise, this looks ready to me.

**Comment by mtesauro** [comment]
@kiblik Just started looking at this - will look deeper later.

But, given how much this touches, I'm thinking its best to merge this after the 2.31.0 release so it has a decent time to 'soak' in the dev branch,

@Maffooch Thoughts on :point_up: ?

File: dojo/templates/base.html [hunk 1]
```diff
@@ -170,12 +170,6 @@
                                           </li>
                                         {% endif %}
                                     {% endif %}
-                                    <li>
-                                        <a href="{% url 'api_v2_schema' %}" target="_blank">
-                                            <i class="fa-solid fa-book fa-fw" title="OpenAPI v2 Docs"></i>
-                                            {% trans "API v2 OpenAPI2 Docs (Deprecated)" %}
-                                        </a>
-                                    </li>
```
[the remark this answers is not in the document]
Comment (kiblik) [reply dojo/templates/base.html:178]: https://github.com/DefectDojo/django-DefectDojo/pull/9469

[the remark this answers is not in the document]
Comment (mtesauro) [reply dojo/templates/base.html:178]: #9469 makes total sense to me - thanks for that! :+1:

**Comment by kiblik** [comment]
Can we approve and merge this now?

**Review by Maffooch — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]

**Pull request merged**